gcmpy/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+# cp (makes exe)
+add_subdirectory(scripts)
+add_subdirectory(yamls)
```

Ahh. I think this is a typo.

```suggestion
add_subdirectory(yaml)
```